src/modules/Jobs/jobCreate/containers/JobCreateForm/index.tsx

```diff
@@ -172,8 +173,8 @@ export const JobCreateForm = () => {
               icon="search-lg"
               options={causesList}
               isSearchable
-              onChange={option => onSelectCause(option)}
-              errors={errors['cause']?.message ? [errors['cause']?.message.toString()] : undefined}
+              onChange={option => onSelectCause(option as OptionType)}
```

Comment:
we should handle `option` type from JobCreateHeader.
I recommend overwriting `onChange` type of SearchDropDown

Reply:
because in most cases, we need `{label, value}` as option.